javascript/.eslintrc.json

```diff
@@ -44,6 +44,8 @@
     "no-unneeded-ternary": ["error", { "defaultAssignment": false }],
     "space-before-function-paren": ["error", "never"],
     "semi": ["error", "always"],
-    "quotes": "off"
+    "quotes": "off",
+    "array-bracket-spacing": ["error", "never"],
```

Comment:
Is this not the default?

Reply:
It is but it's not working. I think this might be messing it up:
https://github.com/standard/eslint-config-standard/blob/c4902d20cab15971932f591d0d8cf7915ade307f/eslintrc.json#L199

Reply:
It works. I verified locally in invoca/backdraft.